snake:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000

  # posiiton controller
  joint1_position_controller:
    type:  effort_controllers/JointPositionController
    joint: link_head_1_joint
    pid: {p: 10, i: 0, d: 0.1}


  joint2_position_controller:
    type:  effort_controllers/JointPositionController
    joint: link_1_2_joint
    pid: {p: 10, i: 0, d: 0.1}


  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_2_3_joint
    pid: {p: 10, i: 0, d: 0.1}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_3_4_joint
    pid: {p: 10, i: 0, d: 0.1}


  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_4_5_joint
    pid: {p: 10, i: 0, d: 0.1}

  joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_5_6_joint
    pid: {p: 10, i: 0, d: 0.1}


  joint7_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_6_7_joint
    pid: {p: 10, i: 0, d: 0.1}


  joint8_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_7_8_joint
    pid: {p: 10, i: 0, d: 0.1}


  joint9_position_controller:
    type: effort_controllers/JointPositionController
    joint: link_8_9_joint
    pid: {p: 10, i: 0, d: 0.1}
